region/crateria/central/Climb Supers Room.json

```diff
@@ -364,7 +370,8 @@
           "canUseIFrames",
           {"spikeHits": 1}
         ]}
-      ]
+      ],
+      "flashSuitChecked": true
```

Comment:
`canUseIFrames` would be another one that may need to be `complexCarry`

Reply:
I don't see the interaction between `canUseIFrames` and a flash suit? Only issue here would be if the player is trying to horizontally damage boost, but there's no reason to do that